Library/Formula/jsawk.rb

```diff
@@ -1,16 +1,18 @@
-require 'formula'
-
 class Jsawk < Formula
-  homepage 'https://github.com/micha/jsawk'
-  url 'https://github.com/micha/jsawk/archive/1.4.tar.gz'
-  sha1 '4f2c962c8a5209764116457682985854400cbf24'
+  homepage "https://github.com/micha/jsawk"
+  url "https://github.com/micha/jsawk/archive/1.4.tar.gz"
+  sha1 "4f2c962c8a5209764116457682985854400cbf24"
 
-  head 'https://github.com/micha/jsawk.git'
+  head "https://github.com/micha/jsawk.git"
 
-  depends_on 'spidermonkey'
+  depends_on "spidermonkey"
 
   def install
-    mv "README.markdown", "README"
```

Comment:
Just out of curiosity, where's the code relating to install README file in homebrew core? I did some searching, no result.

IMHO, if this is a feature in homebrew, we should install README, README.txt, README.md, README.rst, etc automatically.